src/popup/utils/constants.ts

```diff
@@ -428,3 +428,8 @@ export const POPUP_TYPES = [
   POPUP_TYPE_MESSAGE_SIGN,
   POPUP_TYPE_RAW_SIGN,
 ] as const;
+
+export const POPUP_CONNECT_ACCESS = {
+  address: 'address',
+  transactions: 'transactions',
+};
```

Comment:
Having an object of settings is not a bad thing. But for me the question is "why we store things as constants?". And there are 2 reasons:

1. Having a bond - one place to store things.
2. Avoiding errors. So when we use a constant and it's removed or renamed the build process will complain about this and notify us where's the problem. But when we use an object, and it's contents gets changed we don't get any info about this. So imagine a situation where someone has changed `POPUP_CONNECT_ACCESS.address` to `POPUP_CONNECT_ACCESS.hash`. We won't get any errors, because `POPUP_CONNECT_ACCESS.address` will still be valid as `undefined` is a value.

This is why I suggest to create separate constants for each of values. I know this makes the imports more complex, but keeps the app more error-proof.